labs/lab-11/tasks/obj-link-dev/support/Makefile

```
@@ -11,5 +11,5 @@ main: main.o shop.o
 main.o: main.c
 
 clean:
-	-rm -f main *.o
+	-rm -f main main.o
```

```suggestion
	-rm -f main *[!shop].o
```
This removes all `.o`s without `shop.o`.